app/components/common/candidate/candidate_page.html

```diff
@@ -64,4 +56,30 @@
       <odca-money-bar-chart value="{{ amount }}" max="{{ $ctrl.supporting.total_expenditures }}" format="money" color="red" ng-repeat-end></odca-money-bar-chart>
     </div>
   </div>
+
+  <div ng-if="$ctrl.supporting.supporting_money.supporting_expenditures" class="divided-section is-off-screen" when-visible="$ctrl.onVisible">
+    <h3>
+      Expenditures by Independent Committees supporting:
+      {{ $ctrl.supporting.supporting_money.supporting_expenditures | dollar }}
+    </h3>
+    <div class="see-more" >
+      <a class="see-more__content" ui-sref="appMain.committee.supporting({filer_id: $ctrl.candidate.filer_id})">See supporting <span class="text-no-break">committees<span class="fa fa-chevron-right"></span></span></a>
+    </div>
+    <div>
+      <div class="component__heading" ng-repeat-start="(label, amount) in $ctrl.supporting.supporting_money.supporting_by_type">{{ label }}</div>
+      <odca-money-bar-chart value="{{ amount }}" max="{{ $ctrl.supporting.supporting_money.supporting_expenditures }}" format="money" color="red" ng-repeat-end></odca-money-bar-chart>
+    </div>
+  </div>
+
+  <div ng-if="$ctrl.candidate.opposing_money.opposing_expenditures">
+   <h3>
+      Expenditures by Independent Committees opposing:
+      {{ $ctrl.candidate.opposing_money.opposing_expenditures | dollar }}
+   </h3>
+    <div class="see-more" >
+      <a class="see-more__content" ui-sref="appMain.committee.opposing({filer_id: $ctrl.candidate.filer_id})">See opposing <span class="text-no-break">committees<span class="fa fa-chevron-right"></span></span></a>
```

Comment:
So to create this new state, you'll want to copy from [appMain.committee.main](https://github.com/caciviclab/disclosure-frontend/blob/974452ee51e401bc3672cc426277376c583746a9/app/components/appMainModule/committeePageModule/committeePageRoutes.js#L42) and then you'll want to add to it to automatically resolve the opposing data you need. If you look at the [parent state](https://github.com/caciviclab/disclosure-frontend/blob/master/app/components/appMainModule/committeePageModule/committeePageRoutes.js#L17), this is where the data is coming from for the existing contributions page. A quick refactor will make it a little more striaghtforward to copy/paste/edit.